tests/test_config/test_config.py

```diff
@@ -758,9 +782,13 @@ def test_multi_line_copyright(source_date_year, app, monkeypatch):
     ('1970-1990', '1970-{current_year}'),
     ('1970-1990 Alice', '1970-{current_year} Alice'),
 ])
-def test_correct_copyright_year(conf_copyright, expected_copyright, source_date_year):
+def test_correct_copyright_year(
+    conf_copyright: str,
+    expected_copyright: str,
+    source_date_year: str | None,
+) -> None:
     config = Config({}, {'copyright': conf_copyright})
-    correct_copyright_year(_app=None, config=config)
+    correct_copyright_year(..., config=config)  # type: ignore[arg-type]
```

Comment:
Suggetion: we could try to unpick this a bit and make the interface, and then test case, cleaner.  Perhaps `correct_copyright_year` should only accept a single argument, the app to check/correct copyright for?